src/kvp/value.rs

```diff
@@ -0,0 +1,17 @@
+use std::{
+    error::Error,
+    fmt::{Debug, Display},
+    ops::Deref,
+    str::FromStr,
+};
+
+use serde::Serialize;
+
+/// Trait which ensures the value of [`KeyValuePair`][crate::kvp::KeyValuePair]
+/// is validated. Different value implementations should use [`FromStr`] to
+/// parse and validate the value based on the requirements.
+pub trait ValueExt:
```

Comment:
```suggestion
pub trait Value:
```

The `Ext` prefix is used for traits that just add extension methods to other types. If it has more than one impl, it's not an `Ext` trait. If anything needs a bound on the trait, it's probably not an `Ext` trait.

Reply:
Yes, agreed. I will change that.